website/source/docs/commands/lock.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "docs"
+page_title: "Commands: Lock"
+sidebar_current: "docs-commands-lock"
+description: |-
```

Comment:
I think @sethvargo mentioned that the description needs to be a single line of text for SEO reasons. You can use `>` in place of `|-` to do this in YAML.